upstream/CVE-2019-11244.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "CVE-2019-11244",
+  "modified": "2019-04-16T20:14:25Z",
+  "published": "2019-04-16T20:14:25Z",
+  "summary": "`kubectl:-http-cache=\u003cworld-accessible dir\u003e` creates world-writeable cached schema files",
+  "details": "In Kubernetes v1.8.x-v1.14.x, schema info is cached by kubectl in the location specified by --cache-dir (defaulting to $HOME/.kube/http-cache), written with world-writeable permissions (rw-rw-rw-). If --cache-dir is specified and pointed at a different location accessible to other users/groups, the written files may be modified by other users/groups and disrupt the kubectl invocation.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubectl"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.0/AV:L/AC:H/PR:L/UI:R/S:U/C:L/I:L/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "1.8.0"
+            },
+            {
+              "fixed": "1.12.9"
+            },
+            {
+              "introduced": "1.13.0"
+            },
+            {
+              "fixed": "1.13.7"
+            },
+            {
+              "introduced": "1.14.0"
+            },
+            {
+              "fixed": "1.14.3"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "references": [
```

Comment:
As you [found](https://github.com/aquasecurity/vuln-list-k8s/pull/66#discussion_r1343523775) - fixed versions from issue != real fixed versions.
Can you add link to this PR/comment in `references` and maybe update description.

Reply:
Done
https://github.com/aquasecurity/vuln-list-k8s/pull/66/commits/72ebf5fe21929c8aa31addf8a2198125f71f593b